src/webgpu/util/math.ts

```diff
@@ -147,17 +147,16 @@ export function nextAfter(val: number, dir: boolean = true, flush: boolean): Sca
 }
 
 /**
- * @returns ulp(x), the unit of least precision for a specific number as a 32-bit float
+ * @returns ulp(x) for a specific flushing mode
  *
- * ulp(x) is the distance between the two floating point numbers nearest x.
- * This value is also called unit of last place, ULP, and 1 ULP.
- * See the WGSL spec and http://www.ens-lyon.fr/LIP/Pub/Rapports/RR/RR2005/RR2005-09.pdf
- * for a more detailed/nuanced discussion of the definition of ulp(x).
+ * This is the main implementation of oneULP, which is normally what should be
+ * used. This should only be called directly if a specific flushing mode is
+ * required.
  *
  * @param target number to calculate ULP for
  * @param flush should subnormals be flushed to zero
  */
-export function oneULP(target: number, flush: boolean): number {
+export function oneULPImpl(target: number, flush: boolean): number {
```

Comment:
I'm really nitpicking here, but I'm not a massive fan of implementation details leaking into public APIs, even if it's exclusively used for testing.
I'd probably opt for the interface:
`oneULP(target: number, flush?: boolean)`

Where omitting `flush` behaves like `oneULP` and providing it acts like `oneULPImpl`.